README.md

```diff
@@ -391,8 +409,21 @@ When evaluating `function anonymous() { return this }`, we get the invocation co
 
 Getting a reference to `window` is another huge step forward for JSFuck. With the brackets characters, we could only dig in the available objects: numbers, arrays, some functions... With a reference to the global scope, we now have access to any global variable and the inner properties of these globals.
 
----
+### Create regular expression object
+
+You can create regular expression e.g. `/pattern/g` as follows
+
+```js
+[]["fill"]["constructor"]("return RegExp")()("pattern","g")
+```
 
+wchich after remove comma (by using multi-arguments technique without `bind`ing) looks as follows
```

Comment:
"wchich" should be "which"
"after remove comma" should be "after removing the comma"

Reply:
fixed